sdk/communication/communication-chat/recordings/browsers/chatclient_chat_operations/recording_successfully_creates_a_thread.json

```diff
@@ -19,11 +19,11 @@
         "Sec-Fetch-Dest": "empty",
```

Comment:
can you address this open issue to migrate the test recordings while you're making this change? https://github.com/Azure/azure-sdk-for-js/issues/27783

Reply:
Let's keep the larger migration of test recordings to a separate PR where it can be tracked more clearly.

Reply:
Here you go: https://github.com/Azure/azure-sdk-for-js/pull/27872